### DISCUSSION TITLE
How to save a TM30 report programatically instead of rendering to screen.

### DISCUSSION THREAD

**Discussion by ademmler** [colour-science/colour#1294] [Q&A] [answered]
How do I save a TM30 report programatically instead of rendering to screen.
Looking for something like this:
colour.plotting.tm3018.plot_single_sd_colour_rendition_report_full(sd, ..... ).write(???);

Regards Alex

**Comment by thomasmansencal** [comment, marked as answer]
Hi @ademmler,

Try to pass `filename=/path/to/image.png` as a keyword argument.

Cheers,

Thomas

**Comment by ademmler** [comment]
Hi Thomas,

thx for giving me the right hint. Works like a charm.
